File: Version_Android/Calculateur_IMC_App_Java/CalculateurImcJava/app/src/main/java/com/example/calculateurimc/ResultatImcActivity.java
package com.example.calculateurimc;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class ResultatImcActivity extends AppCompatActivity {
    TextView txt_result_imc;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_resultat_imc);
/*

getSupportActionBar().setDisplayHomeAsUpEnabled(true);
va servir à créer la touche de retour sur la barre et il faut aussi
rajouter   ---> android:parentActivityName=".MainActivity"/>
dans le fichier AndroidManifest.xml

 */
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        Intent intent = getIntent();
        String s_recup_imc_txt= intent.getStringExtra("key_send_imc");

        txt_result_imc =(TextView) findViewById(R.id.textView_res);
        txt_result_imc.setText(s_recup_imc_txt);
    }
}

File: Version_Android/Calculateur_IMC__App_Kotlin/CalculateurImcKotlin/app/src/main/java/com/example/calculateurimckotlin/ResultatImcActivity.kt
package com.example.calculateurimckotlin

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.EditText
import android.widget.TextView

class ResultatImcActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_resultat_imc)

        val Intent = intent
        val s_recup_imc_txt : String? = Intent.getStringExtra("key_send_imc")
        val txtView :TextView = findViewById(R.id.textView_res)
        txtView.setText(s_recup_imc_txt)

    }
}